sklearn/cluster/dbscan_.py

```diff
@@ -66,7 +66,9 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
         Note that weights are absolute, and default to 1.
 
     random_state: numpy.RandomState, optional
-        The generator used to shuffle the samples. Defaults to numpy.random.
+        The generator used to shuffle the samples, which affects the cluster
+        numbering and cluster assignments of points that are border points to
+        more than one cluster. Defaults to not shuffling. None.
```

Comment:
This is not the convention used elsewhere; rather, a separate `shuffle` parameter is used.